src/client/components/Sidebar/ListPane/FlagCount.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+// App Imports
+
+// Styles
+const Wrapper = styled.div`
+  display: flex;
+`
+
+const Total = styled.div`
+  font-family: Roboto;
+  font-size: 54px;
+  font-weight: 100;
+  line-height: 72px;
+`
+
+const Label = styled.div`
+  font-family: Roboto;
+  font-size: 18px;
+  width: 86px;
+  height: 46px;
+  line-height: 24px;
+  text-transform: uppercase;
+  padding: 10px;
+`
+
+// Component
+class FlagCount extends React.Component {
+  static propTypes = {
+    flaggedPhrases: PropTypes.array,
```

Comment:
Pretty sure this PropTypes isn't kosher any more. Should be `PropTypes.arrayOf()`, with the actual expected value of each array member further specified (e.g., `PropTypes.arrayOf(PropTypes.string)`). Also, these should be marked as `isRequired` unless we provide a `defaultProps` hash and wait why isn't the linter catching these? Are we using different linters?? 😮

Reply:
lol oh right you're supposed to run linters aren't you.

I'll be honest... absolutely nothing has been linted... I should... probably just make an issue to handle that.